FlightPatternDetection/Models/Airport.cs

```diff
@@ -9,11 +9,11 @@ namespace FlightPatternDetection.Models;
 public class Airport
 {
     [Key]
-    public string ICAO { get; set; }
+    public string? ICAO { get; set; }
 
-    public string Name { get; set; }
+    public string? Name { get; set; }
 
-    public string Country { get; set; }
+    public string? Country { get; set; }
```

Comment:
As for these database-models, it does make sense that we make these nullable, but only because you defined the database tables nullable in your `Database_Creation.sql` script.

But if you made them `not null` in the SQL file, these should not be nullable. In this case, I think we should just disable the nullable warnings for the models file as per this Stackoverflow answer: https://stackoverflow.com/a/67592115/3561728

You don't need to do this, but just a thought.

Reply:
Hmmm yes, I was not sure whether or not we should allow the items in the DB to be null, since it might just bring us some træls:tm: problems in the long run. It was the initial thought to allow them to be null, since I was not sure if we would always be able to get all the needed data/information from the flight data. But yes, we could easily change it to be `not null`

Reply:
Yeah whatever for now. We can always change it when we actually use the DB to see what we actually end up storing. My guess is that we have forgotten an entity or something anyway, so we probably have to change the structure no matter what.